src/icbfms.f

```diff
@@ -27,7 +27,7 @@
 C>
 C> REMARKS:
 C>    THIS ROUTINE CALLS:        IUPM
-C>    THIS ROUTINE IS CALLED BY: RDCMPS   RDTREE   UFDUMP
+C>    THIS ROUTINE IS CALLED BY: RDCMPS   RDTREE   UFBDMP   UFDUMP
```

Comment:
Can you change all function names in documentation to lower-case and with parens?

So:

`THIS ROUTINE IS CALLED BY: rdcmps()   rdtree()   ufdbmp()   ufdump()`

This will cause doxygen to hyperlink that function name, so the user can click on it and go to the documentation for that function.

Emacs has C-x-l to lower-case text in a region, if you use emacs...

Reply:
I was going to get to those eventually, once I really go through each of the subprograms to fully "Doxygen"ate them ;-)  But if you want I'll go ahead and fix those now.